aws/logs_monitoring/steps/handlers/s3_handler.py

```diff
@@ -219,6 +216,25 @@ def parse_service_arn(source, key, bucket, context):
                 return "arn:{}:redshift:{}:{}:cluster:{}:".format(
                     get_partition_from_region(region), region, accountID, clustername
                 )
+
+    if source == "vpc":
+        namesplit = key.split("/")
+        if len(namesplit) >= 4:
+            accountID = namesplit[1]
+            region = namesplit[3]
+
+            filesplit = namesplit[7].split("_")
+
+            if len(filesplit) >= 4:
+                identifier = filesplit[3]
+
+                return "arn:{}:ec2:{}:{}:vpc-flow-log/{}".format(
```

Comment:
Do we want to use this ARN directly on the logs backend side ?

Reply:
Yes, I have a [draft pull-request for this](https://github.com/DataDog/logs-backend/pull/66877/files#diff-9c896f5dc6eec1b4a0af62a7f4750730e0bd0fb4ae5bb8ad3b7942f7e2f9351dR21-R25), but we need to discuss some implementation details.